remote_exec/remote_exec.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+option go_package = "teleport_exec/remote_exec";
+package remote_exec;
+
+//-----------------------------------------------------------------------------
+message StartCommandRequest { string command = 1; }
+
+//-----------------------------------------------------------------------------
+message CommandStatusRequest { string command_id = 1; }
```

Comment:
What will this ID be?

Reply:
When a command it submitted to the system, we generate an UUID for it (pretty much like what docker does with their container ids) and return it to the caller. Then all subsequent operations use that UUID to refer to a specific command on the server. I have mentioned it in https://github.com/kovyrin/teleport-exec/pull/1/files#diff-dab99c095ae6ba83333120109304711e01a03afe34e7a3dde16b08b548d6f1f1R141, but could clarify further if needed.

Reply:
:+1: